sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt

```diff
@@ -31,24 +29,27 @@ internal class SMainActivity : AppCompatActivity() {
         }
 
         binding.sampleCropImage.setOnClickListener {
-            hideButtons(binding)
             supportFragmentManager
                 .beginTransaction()
                 .replace(R.id.container, SCropImageFragment.newInstance())
                 .commit()
         }
 
         binding.sampleCropImageJava.setOnClickListener {
-            hideButtons(binding)
             supportFragmentManager
                 .beginTransaction()
                 .replace(R.id.container, SCropImageFragmentJava.newInstance())
                 .commit()
         }
     }
 
-    private fun hideButtons(binding: ActivityMainBinding) {
-        binding.image.visibility = View.GONE
-        binding.buttons.visibility = View.GONE
+    override fun onBackPressed() {
+        if (supportFragmentManager.findFragmentById(R.id.container) != null) {
```

Comment:
Let's use the view binding, like `binding.container`. We will need to make `binding` value private in the class, like `private var binding: ActivityMainBinding? = null`

Reply:
What do you mean "we have a white background"? Is there a problem displaying the fragment or is it wrong to use a white background?